### ISSUE TITLE
Documentation: Tutorial and example

### ISSUE THREAD

**Issue by pettter** [DCMLab/reductive_analysis_app#1]
Make a minimal use case and example

**Comment by pettter** [comment]
Once again updated the tutorial with the new UI

**Comment by mkarjun** [comment]
It would be nice to have a 'how to section' integrated to the webapp page itself.

**Comment by pettter** [comment]
@mkarjun That's a good idea! Possibly it could be integrated with #224? Or it could be one of those snazzy interactive tutorials where you shade out everything but whatever you're supposed to click on, together with some explanatory dialog boxes..

**Comment by mkarjun** [comment]
@pettter yes that will do ,like integrating interactive tutorials or small animated video to the point tutorial with #224.
